### DISCUSSION TITLE
Default labels cannot be deleted

### DISCUSSION THREAD

**Discussion by yoav-lavi** [community/community#7500] [Pull Requests]
When trying to delete a default label (such as "good first issue"), clicking on "delete" does nothing and the label remains as is

**Comment by Arhell** [comment]
Just checked, labels are remove
![423415234523е4632](https://user-images.githubusercontent.com/26163841/141201921-2bedd1fc-f333-45d5-8415-6798735b646a.jpg)

**Reply by yoav-lavi** [reply]
Checking again

**Reply by yoav-lavi** [reply]
Still not working on my end

**Reply by Arhell** [reply]
Maybe this is not your repository or you do not have permission to do so.

**Reply by yoav-lavi** [reply]
I'm an admin on said repository, I don't think it has to do with permissions

**Reply by Arhell** [reply]
Here's what I found in the documentation
https://docs.github.com/en/issues/using-labels-and-milestones-to-track-work/managing-labels#deleting-a-label
It may help, but as you wrote, it may not

**Reply by Arhell** [reply]
or here
https://support.github.com/

**Reply by yoav-lavi** [reply]
Thanks!

**Reply by minherz** [reply]
I see the same behavior as well. Being administrator on the repo but without similar permissions on the org level, I cannot delete _default_ labels from my repo.
